src/test/avocado-test.ts

```diff
@@ -144,7 +143,26 @@ describe('avocado', () => {
 
   it('backslash', async () => {
     const r = await avocado.avocado({ cwd: 'src/test/backslash', env: {} }).toArray()
-    const expected: unknown = []
+    const expected = [] as const
+    assert.deepStrictEqual(r, expected)
+  })
+
+  it('diamond dependencies', async () => {
+    const r = await avocado.avocado({ cwd: 'src/test/diamond_dependencies', env: {} }).toArray()
+    const expected = [
```

Comment:
as we talked you may want to add a comment that the test is validating for a unique error in this case.